cypress/integration/main-page.test.js

```diff
@@ -0,0 +1,35 @@
+import 'cypress-file-upload';
```

Comment:
We need to mock response from server because we will run this saga on our test server.

Reply:
fixed